Signal/src/ViewControllers/ColorPickerViewController.swift

```diff
@@ -166,6 +191,58 @@ class ColorPickerView: UIView, ColorViewDelegate {
         return headerView
     }
 
+    private func updateMockConversationView() {
+        conversationStyle.viewWidth = max(bounds.size.width, kMinimumConversationWidth)
+        mockConversationView.subviews.forEach { $0.removeFromSuperview() }
+
+        // outgoing
+        outgoingMessageView = OWSMessageBubbleView(forAutoLayout: ())
+        outgoingMessageView.prepareForReuse()
```

Comment:
I'm surprised that `prepareForReuse` is necessary here since we just made the view.

Reply:
Thanks for catching that - I added it during some earlier debugging. I've removed it.